modules/ROOT/pages/amazon/amazon-sqs-connector-design-center.adoc

```diff
@@ -0,0 +1,69 @@
+= Amazon SQS Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the
+xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Configure the target component for your app.
+
+For information on Flow Designer field values, see
+the xref:amazon/amazon-sqs-connector-reference.adoc[Amazon SQS Connector Reference].
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click on the trigger card.
+. Search for the connector.
+. Select the connector name.
+. Select the *Receive messages* operation to use it as a source. (You can also use the HTTP Connector or Scheduler component as a source.)
+. Click *Click here to set it up*.
+. Enter values for these fields:
++
+[%header,cols="30s,70a",width=80%]
+|===
+|Parameter |Description
```

Comment:
QUESTION:
Does it matter if there is a space or not before the term that follows the pipe symbol (|) in table column formatting? For example, "|Description" vs. "| Description"? Note that the following entries are inconsistent.

Reply:
It does not matter